application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -385,8 +384,7 @@ public function set_survey_properties($sSessionKey, $iSurveyID, $aSurveyData)
             if (Permission::model()->hasSurveyPermission($iSurveyID, 'surveysettings', 'update')) {
                 // Remove fields that may not be modified
                 unset($aSurveyData['sid']);
-                //unset($aSurveyData['owner_id']);
-                unset($aSurveyData['active']);
+                if (@$aSurveyData['active'] == 'Y') unset($aSurveyData['active']);
```

Comment:
IMHO, the stfu operator should be avoided as much as possible.
is there any reason why you use it?

Reply:
My clumsy way to keep it shorter than ```if (isset($aSurveyData['active']) && $aSurveyData['active'] == 'Y')```

Reply:
And eventually these conditions might need to be recursive if done properly.